libnmstate/schema.py

```diff
@@ -288,5 +288,20 @@ class OVSBridge(object):
 
     PORT_SUBTREE = 'port'
     PORT_NAME = 'name'
-    PORT_VLAN_MODE = 'vlan-mode'
-    PORT_ACCESS_TAG = 'access-tag'
+
+    class Port:
+        VLAN_SUBTREE = 'vlan'
```

Comment:
Why is the NAME not in here?

Reply:
This is handled by the constants changes PR/s, not this one.
I will move it in a separate one.